Resize VirtualBox disk to 300gb

ldub opened the pull request:
I've put together a script to help fix #50. I'm new to NixOps and I'm not sure how to make NixOps run these commands for me. If can give me a quick pointer for how this is supposed to be done, I'll update this PR.

Example run:
```
$ ./scripts/vbox-resize-disk1.sh
++ DEPLOYMENT=bitcoin-node
++ MACHINE=bitcoin-node
+++ nixops export -d bitcoin-node
+++ nix-shell -p jq --command 'jq -r '\''..|."virtualbox.disks"?|select(.!=null)'\'' | jq -r .disk1.path'
++ DISK_FILE='/Users/lev/VirtualBox VMs/nixops-81b16311-9af4-11e9-abb6-d0817ad44941-bitcoin-node/disk1.vdi'
++ nixops stop -d bitcoin-node
bitcoin-node> shutting down... [running]
bitcoin-node> [running] Connection to 192.168.56.115 closed by remote host.

bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [running]
bitcoin-node> [poweroff] ++ VBoxManage modifyhd --resize 266240 '/Users/lev/VirtualBox VMs/nixops-81b16311-9af4-11e9-abb6-d0817ad44941-bitcoin-node/disk1.vdi'
0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
++ nixops start -d bitcoin-node
bitcoin-node> starting...
bitcoin-node> Waiting for VM "nixops-81b16311-9af4-11e9-abb6-d0817ad44941-bitcoin-node" to power on...
bitcoin-node> VM "nixops-81b16311-9af4-11e9-abb6-d0817ad44941-bitcoin-node" has been successfully started.
bitcoin-node> waiting for IP address... 192.168.56.115
bitcoin-node> waiting for SSH........................................................................
++ nixops ssh -d bitcoin-node bitcoin-node -- '(echo d; echo n; echo p; echo; echo; echo; echo w; ) | fdisk /dev/sda'

set -ex
Welcome to fdisk (util-linux 2.33.1).
Changes will remain in memory only, until you decide to write them.
set -ex
Be careful before using the write command.


Command (m for help): Selected partition 1
Partition 1 has been deleted.

Command (m for help): Partition type
   p   primary (0 primary, 0 extended, 4 free)
   e   extended (container for logical partitions)
Select (default p): Partition number (1-4, default 1): First sector (2048-545259519, default 2048): Last sector, +/-sectors or +/-size{K,M,G,T,P} (2048-545259519, default 545259519):
Created a new partition 1 of type 'Linux' and of size 260 GiB.

Command (m for help): Partition #1 contains a ext4 signature.

The partition table has been altered.
Syncing disks.

++ nixops reboot -d bitcoin-node
bitcoin-node> rebooting...
Connection to 192.168.56.115 closed by remote host.
bitcoin-node> waiting for the machine to finish rebooting.....[down].............................................................................[up]
bitcoin-node> uploading key ‘ssl_certificate’...
bitcoin-node> uploading key ‘lightning-charge-api-token-for-nanopos’...
bitcoin-node> uploading key ‘ssl_certificate_key’...
bitcoin-node> uploading key ‘bitcoin-rpcpassword’...
bitcoin-node> uploading key ‘lightning-charge-api-token’...
bitcoin-node> uploading key ‘spark-wallet-login’...
++ nixops ssh -d bitcoin-node bitcoin-node -- resize2fs /dev/sda1
resize2fs 1.44.5 (15-Dec-2018)
Filesystem at /dev/sda1 is mounted on /; on-line resizing required
old_desc_blocks = 32, new_desc_blocks = 33
The filesystem on /dev/sda1 is now 68157184 (4k) blocks long.

++ nixops ssh -d bitcoin-node bitcoin-node -- df -h
Filesystem                Size  Used Avail Use% Mounted on
devtmpfs                  198M     0  198M   0% /dev
tmpfs                     2.0G     0  2.0G   0% /dev/shm
tmpfs                     988M  3.6M  984M   1% /run
tmpfs                     2.0G  360K  2.0G   1% /run/wrappers
/dev/disk/by-label/nixos  256G   48G  198G  20% /
tmpfs                     2.0G     0  2.0G   0% /sys/fs/cgroup
tmpfs                     395M     0  395M   0% /run/user/0
```

jonasnick commented on docs/install.md:
````diff
@@ -115,7 +115,14 @@ You can also build Nix from source by following the instructions at https://nixo
+	./scripts/vbox-resize-disk1.sh
+	```
+
+	NixOps provides a virtualbox disk thats 50gb in size, but we need more than that to house the Bitcoin blockchain. Make sure to run this from within your nix-shell.
````
nit: Can you mention the size it's changed to? Or even add an argument to `./scripts/vbox-resize-disk1.sh` with the size in GB

ldub replied:
done.

jonasnick commented on scripts/vbox-resize-disk1.sh:
```diff
@@ -0,0 +1,24 @@
+set -ex
```
Can you rename the scripts directory to `contrib`? Also, I think this script needs to be made executable.

ldub replied:
done.

ldub commented:
Changed it to add the shebang. I've never used that feature of nix but now I realize how useful that is!

Also added some configurability to the script.
